spec/migrations/20160310170333_add_service_ancestry_spec.rb

```diff
@@ -0,0 +1,50 @@
+require "spec_helper"
+require_migration
+
+describe AddServiceAncestry do
+  let(:service_stub) { migration_stub(:Service) }
+
+  migration_context :up do
+    it "updates tree" do
+      s21  = service_stub.create!
```

Comment:
Can you draw like an ASCII tree comment here so it's clear which s relates to which other s?